django/contrib/postgres/indexes.py

```diff
@@ -1,8 +1,26 @@
 from __future__ import unicode_literals
 
-from django.db.models import Index
+from django.db.models.indexes import Index
 
-__all__ = ['GinIndex']
+__all__ = ['BRINIndex', 'GinIndex']
+
+
+class BRINIndex(Index):
+    """
+    Support for the BRIN index introduced in PostgreSQL 9.5. There's an optional
+    parameter ``pages_per_range`` which is passed as an argument.
+    """
+    suffix = 'brin'
+
+    def __init__(self, *args, **kwargs):
+        self.pages_per_range = kwargs.pop('pages_per_range', None)
+        return super(BRINIndex, self).__init__(*args, **kwargs)
+
+    def create_sql(self, model, schema_editor):
+        sql = super(BRINIndex, self).create_sql(model, schema_editor, using=' USING brin')
+        if self.pages_per_range is not None:
+            sql += " WITH (pages_per_range = {})".format(self.pages_per_range)
```

Comment:
You might append this to the `using` sql passed to `super()` to fix the test failure on Jenkins (due to usage of TABLESPACE).